### DISCUSSION TITLE
1.2.4 release (and 1.2.5)

### DISCUSSION THREAD

**Discussion by sargonas** [Chia-Network/chia-blockchain#8243] [Announcements]
Hi all! We excited to announce the release of 1.2.4 this week. This is primarily a maintenance release with a few minor additions as well.

https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.4

**Comment by sargonas** [comment]
Known Issue(s):

If you are having problems running the GUI install script and getting `AttributeError:`, our apologies to you all. The install script is running `chia version` to get the version number to then feed into the GUI installer, however there is a problem we found with the parsing due to `chia version` returning some SSL permission issues in this build

To fix this, after the core chia install you can then run `chia init --fix-ssl-permissions` to fix those permissions, and if you then run `chia version` after that and get a version number with no permissions issues, you'll know you are good to go. You can then move on to the next step and run the GUI install script as normal!

For more info you can track the issue around this here: https://github.com/Chia-Network/chia-blockchain/issues/8234

**Reply by sargonas** [reply]
FYI release 1.2.5 has just been released specifically to address this issue, it's the only change from 1.2.4.

https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.5

**Comment by Dnepr-LeO** [comment]
Hi! I have a problem with install.
When launched NVIDIA processes (drivers for videocard) I receive error for install CHIA client.
If I stopped NVIDIA processes I have install sucesfull.
